ide/wg_Find.ml

```diff
@@ -115,6 +116,8 @@ class finder name (view : GText.view) =
           let next = view#buffer#get_iter_at_mark (`MARK stop_mark) in
           let () = view#buffer#delete_mark (`MARK start_mark) in
           let () = view#buffer#delete_mark (`MARK stop_mark) in
+          let _ = incr count in
+          let _ = Ideutils.flash_info ("Replaced " ^ (string_of_int !count) ^ " occurrences") in
```

Comment:
Shouldn't this message be placed at line 107 (the `None` case)?

Reply:
Yes.